src/views/MainPage/components/IntroCardList/IntroCard/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { motion } from 'framer-motion';
+import { ReactComponent as IcEllipseBlue } from '@src/assets/icons/ic_ellipse_blue.svg';
+import { IntroCardType } from '@src/lib/types/main';
+import * as S from './style';
+
+interface IntroCardProps {
+  card: IntroCardType;
+}
+
+export default function IntroCard({ card }: IntroCardProps) {
+  const circleDraw = {
+    hidden: { opacity: 0 },
+    visible: {
+      opacity: 1,
+      transition: {
+        type: 'spring',
+        damping: 20,
+        stiffness: 120,
+        opacity: { delay: 0.2, duration: 0.3 },
+      },
+    },
+  };
+
+  const lineDraw = {
+    hidden: { pathLength: 0, opacity: 0 },
+    visible: {
+      pathLength: 1,
+      opacity: 1,
+      transition: {
+        pathLength: { delay: 0.6, type: 'spring', duration: 2.5, bounce: 0 },
+        opacity: { delay: 0.6, duration: 0.2 },
+      },
+    },
+  };
+
+  const cardDraw = {
+    hidden: { x: -10, opacity: 0 },
+    visible: {
+      x: 0,
+      opacity: 1,
+      transition: {
+        type: 'spring',
+        damping: 20,
+        stiffness: 120,
+        x: { delay: 0.3, duration: 0.8 },
+        opacity: { delay: 0.3, duration: 1 },
+      },
+    },
+  };
+
+  return (
+    <S.IntroCard>
+      <S.Box>
+        <S.MotionCircle
+          initial="hidden"
+          whileInView="visible"
+          variants={circleDraw}
+          viewport={{ amount: 0.4 }}
+        >
+          <IcEllipseBlue />
+        </S.MotionCircle>
+        {card.id !== 3 && (
+          <S.MotionLine
+            viewBox="0 0 20 calc(100vh + 2px)"
+            initial="hidden"
+            whileInView="visible"
+            viewport={{ amount: 0 }}
+          >
+            <motion.line x1="10" y1="10" x2="10" y2="100vh" stroke="#d7f5ff" variants={lineDraw} />
```

Comment:
우왕 svg에 vh단위로 속성을 줘도 동작하나요 ..!?!? 처음 보아서 여쭙습니다!

Reply:
답변 드리려고 계속 찾아봤는데 제가 잘 동작하는 것처럼 느꼈던 이유가 아래와 같네요....!
> The <svg> element is a sort of a viewport itself with its own bounding box, the viewBox, the defines the position and dimension of the SVG. As such, a value like 100vh will evaluate to 100% of the SVG height instead of 100% of the viewport height.

제가 viewbox에 vh를 사용했고 y2에 쓴 100vh가 100%처럼 작동해서 제대로 동작하는 것처럼 느낀 거 같습니다